### PR TITLE
Home page2

### PR DESCRIPTION
**Pull request by pAditya198**
Screenshot (84)

Resolved issue #33 also fixed x-overflow on this page.
Padding issue fixed.

### CODE REVIEW

**Comment by pAditya198** [comment]
@aashutoshrathi @parth181195 pls review.

**Comment by pAditya198** [comment]
Screenshot (85)


resolved issue #35 

@parth181195 @aashutoshrathi @samagragupta please review

**Comment by pAditya198** [comment]
Okay I'll take care about that

File: src/app/components/home-page/home-page.component.html [hunk 1]
```diff
@@ -223,14 +223,30 @@ <h3 class="sub-title Previous">Previous Rockstars Speakers</h3>
   <div id="subscribe">
     <div class='row center-lg center-xs center-md middle-lg middle-md middle-sm
       middle-xs'>
-      <div class="col-xs-12 col-md-12" style='height: 150px'>
-        <div class="sub-title" style='line-height:150px;font-size: 2em'>
-          Get notified about important conference updates
+      <div class="col-xs-12 col-md-12" style='height: max-content;'>
+        <br>
+        <div class="sub-title" style='line-height:50px;font-size: 2em'>
+          Contact US
```
Comment (aashutoshrathi) [inline src/app/components/home-page/home-page.component.html]: *Us

File: src/app/components/home-page/home-page.component.html [hunk 2]
```diff
@@ -223,14 +223,30 @@ <h3 class="sub-title Previous">Previous Rockstars Speakers</h3>
...
+        </div>
+        <div class="sub-title" style='line-height:70px;font-size: 1.4em'>
+          gdggandhinagar@gmail.com
+        </div>
+        <div class="sub-title" style='line-height:70px;font-size: 2.2em'>
+          Subscribe to Us for Updates
```
Comment (aashutoshrathi) [inline src/app/components/home-page/home-page.component.html]: Subscribe for Updates

**Comment by pAditya198** [comment]
Screenshot (87)

speaker dialog overflow-x fixed

Screenshot (88)

Contact Us fixed

File: src/app/components/home-page/home-page.component.html (hunk 2 above)
Comment (pAditya198) [reply src/app/components/home-page/home-page.component.html]: fixed

File: src/app/components/home-page/home-page.component.html (hunk 1 above)
Comment (pAditya198) [reply src/app/components/home-page/home-page.component.html]: fixed

File: src/app/components/home-page/home-page.component.html [hunk 3]
```diff
@@ -223,14 +223,35 @@ <h3 class="sub-title Previous">Previous Rockstars Speakers</h3>
   <div id="subscribe">
     <div class='row center-lg center-xs center-md middle-lg middle-md middle-sm
       middle-xs'>
-      <div class="col-xs-12 col-md-12" style='height: 150px'>
-        <div class="sub-title" style='line-height:150px;font-size: 2em'>
-          Get notified about important conference updates
+      <div class="col-xs-12 col-md-12" style='height: max-content;'>
+        <br>
+        <div class="sub-title" style='line-height:50px;font-size: 2em'>
```
Comment (aashutoshrathi) [inline src/app/components/home-page/home-page.component.html]: Make this bold

File: src/app/components/home-page/home-page.component.html [hunk 4]
```diff
@@ -223,14 +223,35 @@ <h3 class="sub-title Previous">Previous Rockstars Speakers</h3>
...
+          Contact Us
+        </div>
+        <div class="sub-title" style='line-height:70px;font-size: 1.4em'>
+          gdggandhinagar@gmail.com
+        </div>
+        <div class="sub-title" style='line-height:70px;font-size: 2.2em'>
+          Subscribe for Updates
```
Comment (aashutoshrathi) [inline src/app/components/home-page/home-page.component.html]: Also, make this bold

File: src/app/components/home-page/home-page.component.html [hunk 5]
```diff
@@ -223,14 +223,35 @@ <h3 class="sub-title Previous">Previous Rockstars Speakers</h3>
...
-      <div class="col-xs-12 col-md-12" style='height: 150px'>
-        <div class="sub-title" style='line-height:150px;font-size: 2em'>
-          Get notified about important conference updates
+      <div class="col-xs-12 col-md-12" style='height: max-content;'>
+        <br>
+        <div class="sub-title" style='line-height:50px;font-size: 2em'>
+          Contact Us
+        </div>
+        <div class="sub-title" style='line-height:70px;font-size: 1.4em'>
+          gdggandhinagar@gmail.com
```
Comment (aashutoshrathi) [inline src/app/components/home-page/home-page.component.html]: Make it clickable.
and `href="mailto:$email"`

File: src/app/components/home-page/home-page.component.html (hunk 3 above)
Comment (pAditya198) [reply src/app/components/home-page/home-page.component.html]: what?

File: src/app/components/home-page/home-page.component.html (hunk 5 above)
Comment (pAditya198) [reply src/app/components/home-page/home-page.component.html]: $email? or email address

File: src/app/components/home-page/home-page.component.html (hunk 3 above)
Comment (pAditya198) [reply src/app/components/home-page/home-page.component.html]: class?

**Comment by pAditya198** [comment]
Screenshot (89)

**Pull request merged**